client_src/js/printerManagementRunner.js

```diff
@@ -3002,24 +3016,29 @@ class dashUpdate {
               let updatePluginButton = document.getElementById(
                 `octoprintPluginUpdate-${printer._id}`
               );
-
               if (printer.updateAvailable.octoPrintUpdate.updateAvailable) {
```

Comment:
- Why is this property so nested?
- Why does it contain `updateAvailable` twice?

Reply:
Because raisins. 

Could be reworked. Only used here in the client. 

The updateAvailable object contains both OP and OP plugin updates.